src/Generators/Webserver/Database/DatabaseGenerator.php

```diff
@@ -66,6 +66,9 @@ protected function driver(array $config)
 
         switch ($driver) {
             case 'pgsql':
+                if ($this->mode === Connection::DIVISION_MODE_SEPARATE_SCHEMA) {
+                    return new Drivers\PostgresSchema;
+                }
```

Comment:
Can we make the above a single line on the next return statement:

```php
return $this->mode === Connection::DIVISION_MODE_SEPARATE_SCHEMA ? new Drivers\PostgresSchema : new Drivers\PostgreSQL;

Reply:
Originally I was going to do the same, but then I realised that it would be too wide for my taste, well beyond 80 columns, but even beyond 120 columns. But anyway, if you prefer the ternary style, it's fine too.

Reply:
And, yes I've run the whole test suite (with skips) and tested manually in a real laravel installation.